.travis.yml

```diff
@@ -28,17 +28,18 @@ before_install:
   - source $HOME/google-cloud-sdk/path.bash.inc || ls -al $HOME/google-cloud-sdk
   - command -v gcloud
   - gcloud version
+  # eslint 6.0.1 requires specific node versions.
+  - nvm install 10.13.0
   - node -v
-  - gulp -v
 
 install:
-  - yarn add global gulp-cli
   - gcloud components install app-engine-python
-  - pip install -r requirements.txt
-  - pip install -r test-requirements.txt
-  # eslint requires node ^6.14.0 || ^8.10.0 || >=9.10.0 -- but doesn't actually need it.
-  # trusty (as used on travis) comes with 8.9.1
-  - yarn install --ignore-engines
+  - pip install --upgrade -r test-requirements.txt
+  - yarn install
+
+before_script:
+  - yarn global add gulp-cli
+  - gulp -v
```

Comment:
we can remove this now

Reply:
We display versions of all sorts of things for the benefit of the logging -- in theory helps debug travis issues. I'd rather leave the `gulp -v` in.